tutorials/gui/bbcode_in_richtextlabel.rst

```diff
@@ -41,51 +41,53 @@ markup. All changes to the text must be done in the BBCode parameter.
 
 .. note::
 
-    For BBCode tags such as ``[b]`` (bold) or ``[i]`` (italics) to work you must
-    set up custom fonts for the RichTextLabel node first.
+    For BBCode tags such as ``[b]`` (bold), ``[i]`` (italics) or ``[code]`` to
+    work, you must set up custom fonts for the RichTextLabel node first.
+
+    There are no BBCode tags to control vertical centering of text yet.
```

Comment:
I guess this can be done using RichTextEffects :P
But that's outside the scope of this note.